### PR TITLE
Fetch unit types items from klass

### PR DESCRIPTION
**Pull request by Jorgen-5**
- Updated to use new datadoc model
- Added multiple languages for unit types
- Added unit types to dropdown
- Saved unit codes to the model
- Added tests for multiple languages

### CODE REVIEW

File: src/datadoc/backend/datadoc_metadata.py [hunk 1]
```diff
@@ -49,31 +48,29 @@ def __init__(
     ) -> None:
         """Read in a dataset if supplied, otherwise naively instantiate the class."""
         self._statistic_subject_mapping = statistic_subject_mapping
-        self.dataset_path = dataset_path
+        # self.dataset_path = dataset_path
```
Comment (JanhSander) [inline src/datadoc/backend/datadoc_metadata.py]: Can be removed?

File: src/datadoc/backend/unit_types.py [hunk 2]
```diff
@@ -73,7 +153,7 @@ def _get_classification_dataframe_if_loaded(self) -> bool:
         return False
 
     @property
-    def classifications(self) -> list[str]:
+    def classifications(self) -> list[UnitType]:
         """Getter for primary subjects."""
         self._get_classification_dataframe_if_loaded()
```
Comment (mmwinther) [inline src/datadoc/backend/unit_types.py:158]:
```suggestion
        if not self._classifications:
            self._get_classification_dataframe_if_loaded()
```

We only want to process this once, not every time the classifications are accessed.

File: src/datadoc/config.py [hunk 3]
```diff
@@ -128,3 +128,8 @@ def get_dapla_service() -> DaplaService | None:
 def get_oidc_token() -> str | None:
     """Get the JWT token from the environment."""
     return _get_config_item("OIDC_TOKEN")
+
+
+def get_unit_code() -> int | None:
+    """Get the unit code for the correct classifications from the python klass package."""
```
Comment (mmwinther) [inline src/datadoc/config.py:134]:
```suggestion
    """The code for the Unit Type code list in Klass."""
```

File: tests/resources/utdata/befolkning/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json
Comment (mmwinther) [inline tests/resources/utdata/befolkning/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json:1]: This file should be deleted I believe, I removed it in my test refactoring PR

File: tests/resources/statistikk/befolkning/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json
Comment (mmwinther) [inline tests/resources/statistikk/befolkning/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json:1]: This file should be deleted I believe, I removed it in my test refactoring PR

File: tests/resources/inndata/befolkning/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json
Comment (mmwinther) [inline tests/resources/inndata/befolkning/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json:1]: This file should be deleted I believe, I removed it in my test refactoring PR

**Review by mmwinther — CHANGES_REQUESTED** [review]
Great work! I found a few small things that need changing, nothing major though.

**Review by JanhSander — CHANGES_REQUESTED** [review]

**Review by mmwinther — APPROVED** [review]
Nice work!

**Review by JanhSander — APPROVED** [review]

**Pull request merged**